static/css/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+    font-family: 'Turret Road';
+    src: url('../fonts/Turret_Road/TurretRoad-Regular.ttf') format('truetype');
+}
+
+@font-face {
+    font-family: 'Turret Road';
+    font-weight: 700;
+    src: url('../fonts/Turret_Road/TurretRoad-Bold.ttf') format('truetype');
+}
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-size: 62.5%;
+    color: #000500;
+    font-family: 'Turret Road', monospace;
+}
+
+.robotHeader {
+    height: 10vh;
+}
+
+.container {
+    min-height: 85vh;
+}
+
+.robotFooter {
+    height: 5vh;
+}
+
+.robotHeader,
+.robotFooter {
+    background-color: #117a8b52;
+}
+
+.robotHeader h1{
+    margin: 0;
+    padding-top: 2vh;
+    font-size: 2rem;
```

Comment:
Should the title Robot Stores be bigger? It is the same font size as the titles for each product and so it kind of disappears in the web page.

Reply:
Also the prices could have a larger font size? They aren't very noticable either in comparison to the large text for each of the products' titles :)